test/util_test.dart

```diff
@@ -614,8 +614,6 @@ void overReactExample() {}''';
           expect(
               allDescendantsOfType<SimpleIdentifier>(node).toList(),
               unorderedEquals([
-                isA<SimpleIdentifier>()
-                    .having((node) => node.name, 'name', 'Foo'),
```

Comment:
`Foo` is now a `NamedType`, and also doesn't show up in this list since it doesn't get visited anymore.

So, I think it's safe to just stop expecting it here.